calendarStyle.css

```
@@ -0,0 +1,109 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "El Messiri", sans-serif;
+}
+
+html{
+    font-size: 62.5%;
+}
+
+.container {
+    color:#E85A4F;
+    background-color: #8E8D8A;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.calendar {
+    width: 40.5rem;
+    height: 42.8rem;
+    background-color: #EAE7DC;
+    background-color: rgba(234, 231, 220, 0.9); 
```

```suggestion
    background-color: rgba(234, 231, 220, 0.9);
```

불필요한 공백이 있습니다.
이렇게 눈에 잘 보이지 않는 공백의 경우, 일일히 체크해서 삭제하는게 번거롭다면 vscode의 Trailing Spaces와 같은 익스텐션을 사용하시는 것도 방법이 될 수 있습니다.

- https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces